conan/tools/conf/conf_aggregator.py

```diff
@@ -0,0 +1,30 @@
+import os
+import json
+from collections import OrderedDict
+
+from conans.util.files import save
+
+
+class ConfAggregator:
+    """Generator that accumulate the configs from the build-require dependencies"""
+    filename = "conf_aggregator.json"
+
+    def __init__(self, conanfile):
+        self._conanfile = conanfile
+        self.data = OrderedDict()
+        # Same behavior than _receive_conf, direct build requires only
+        for dep in self._conanfile.dependencies.direct_build.values():
```

Comment:
I think this is the key, so far there is only 1 conf that makes sense to accumulate in a list, out of 30 that doesn't make sense at all. So this aggregator will be doing the right thing for 1 conf, but the wrong one for 29, so not very clear to me the value. Maybe more than a general Conf aggregator would be a utility that allows to aggregate the specified conf value, just a single one?

Reply:
Might be, but still unclear to me that this is the way given my previous comment on the PR.